runtime/src/accounts/mod.rs

```diff
@@ -996,12 +999,9 @@ mod tests {
 
         let loaded_accounts = load_accounts_aux_test(tx, &accounts, &mut error_counters);
 
-        assert_eq!(error_counters.invalid_program_for_execution, 1);
+        assert_eq!(error_counters.invalid_program_for_execution, 0);
         assert_eq!(loaded_accounts.len(), 1);
-        assert_eq!(
-            loaded_accounts[0],
-            (Err(TransactionError::InvalidProgramForExecution), None,)
-        );
+        assert!(loaded_accounts[0].0.is_ok());
```

Comment:
This seems to be an un-feature-gated change to consensus.

Reply:
yeah. this need to be feature-gated.

Reply:
ok. let me think about it. 

Another question: 

https://github.com/solana-labs/solana/blob/master/sdk/src/transaction_context.rs#L1082

With the change of is_executable() on BorrowedAccount, this seems not correct any more? upgradable account can_data_be_changed?